### PR TITLE
Add support to fetch CQL Server AUH data

### PR DESCRIPTION
**Pull request by abhinab-yb**

### CODE REVIEW

File: src/yb/master/master_tserver.h [hunk 1]
```diff
@@ -80,7 +80,9 @@ class MasterTabletServer : public tserver::TabletServerIf,
 
   std::vector<yb::util::WaitStateInfoPtr> GetThreadpoolWaitStates() const override;
 
-  std::vector<WaitStateInfoPB> ActiveUniverseHistory() const override;
+  void ActiveUniverseHistory(tserver::PgActiveUniverseHistoryResponsePB* resp) const override {}
+
+  void GetCQLServerMessenger(tserver::CQLServerMessenger messenger) override {};
```
Comment (hbhanawat) [inline src/yb/master/master_tserver.h:85]: Do we need these changes?

File: src/yb/yql/pggate/util/ybc_stat.cpp [hunk 2]
```diff
@@ -88,6 +88,9 @@ ybcstat_get_wait_event_type(uint32_t wait_event_info) {
         case YB_FLUSH_AND_COMPACTION:
             event_type = "Flush and Compaction";
             break;
+        case YB_CQL_WAIT_STATE:
```
Comment (hbhanawat) [inline src/yb/yql/pggate/util/ybc_stat.cpp:91]: Should we have a separate wait event component for YCQL just like PG?
We can then have the following values
wait event component: YCQL
wait event Type: Query processing 
wait event:   Parse/Analyze/Execute

File: src/yb/yql/pggate/util/ybc_stat.cpp [hunk 3]
```diff
@@ -88,6 +88,9 @@ ybcstat_get_wait_event_type(uint32_t wait_event_info) {
...
+            event_type = "CQL Server";
+            break;
```
Comment (hbhanawat) [inline src/yb/yql/pggate/util/ybc_stat.cpp:93]: I don't see wait events for DMLRead and DMLWrite i.e. when CQLServer has sent an RPC to tserver.

File: src/yb/yql/cql/ql/ql_processor.cc [hunk 4]
```diff
@@ -428,6 +431,7 @@ void QLProcessor::ExecuteAsync(const ParseTree& parse_tree, const StatementParam
       return;
     }
   }
+  SET_WAIT_STATUS(util::WaitStateCode::Execute);
```
Comment (hbhanawat) [inline src/yb/yql/cql/ql/ql_processor.cc:434]: When is this wait state cleared?

File: src/yb/master/master_tserver.h (hunk 1 above)
Comment (abhinab-yb) [reply src/yb/master/master_tserver.h:85]: CQL server is not accessible from TServer, so the second function installs a callback to get the CQL Server callback. 

The first function, I think that makes it a little more efficient

File: src/yb/yql/pggate/util/ybc_stat.cpp (hunk 2 above)
Comment (abhinab-yb) [reply src/yb/yql/pggate/util/ybc_stat.cpp:91]: Makes sense, I'll update it

File: src/yb/yql/pggate/util/ybc_stat.cpp (hunk 3 above)
Comment (abhinab-yb) [reply src/yb/yql/pggate/util/ybc_stat.cpp:93]: Adding these

File: src/yb/master/master_tserver.h (hunk 1 above)
Comment (hbhanawat) [reply src/yb/master/master_tserver.h:85]: Didn't get it. These calls are in master. Is it because of inheritance that you need them here?

Comment (abhinab-yb) [reply src/yb/master/master_tserver.h:85]: Yeah, these are also included in master because of inheritance

Comment (hbhanawat) [reply src/yb/master/master_tserver.h:85]: The base class should have this function rather than all the inheriting classes.

Comment (abhinab-yb) [reply src/yb/master/master_tserver.h:85]: The base class has only virtual functions, so there needs to be a no-op definition here. The other examples are similar too

**Pull request closed without merging**